src/metamodel/concerto_metamodel_1_0_0.rs

```diff
@@ -190,12 +191,17 @@ pub struct Decorator {
    pub location: Option<Range>,
 }
 
-#[derive(Debug, Serialize, Deserialize)]
+#[derive(Debug, Clone, Serialize, Deserialize)]
```

Comment:
I guess the `Clone` trait is needed here. However, we should add this to the Rust target for concerto-codegen, instead of modifying this file.